virttest/libvirt_vm.py

```diff
@@ -2834,13 +2834,18 @@ def install_package(self, name, ignore_status=False):
         :param name: Name of package to be installed
         """
         session = self.wait_for_login()
-        if not utils_package.package_install(name, session):
-            if not ignore_status:
-                session.close()
+        try:
+            if not utils_package.package_install(name, session):
```

Comment:
Also need to add condition judgment for ignore_status as before as if install fail and no exception, no need to raise error as before

Reply:
I assume if install fail , it need intentionedly raise error whether it throw exception itself or not.

Reply:
if install failed without a exception raised, I'm still raising a virt_vm.VMError in following untouched code as before. 
So if install failed OR exception raised by utils_package.package_install, a exception will always caught by the except clause.

Reply:
So you mean to remove the following
```
raise virt_vm.VMError("Installation of package %s failed" %
                                       name)
```
But instead to add `if ignore_status` in the try clause?